paasta_tools/long_running_service_tools.py

```diff
@@ -220,6 +218,16 @@ def get_registrations(self) -> List[str]:
 
         return registrations or [compose_job_id(self.service, self.instance)]
 
+    def get_invalid_registrations(self) -> List[str]:
+        registrations = self.config_dict.get("registrations", [])
+        invalid_registrations: List[str] = []
+        for registration in registrations:
+            try:
+                decompose_job_id(registration)
```

Comment:
If you wanted, I think you could go further and check that the first element of the return value of `decompose_job_id` is a valid service name, though I'm not 100% sure that's a strict requirement right now.

Reply:
Yeah I guess I just want to ship this if that is ok